t/t1103-apidisconnect.t

```diff
@@ -32,5 +34,14 @@ test_expect_success 'multi-node kvs watcher gets disconnected on client exit' '
 	${FLUX_BUILD_DIR}/t/kvs/watch_disconnect $SIZE
 '
 
+test_expect_success 'module watcher gets disconnected on module unload' '
+	before_watchers=`flux module stats --parse "watchers" kvs-watch` &&
+	echo "waiters before loading module: $before_watchers" &&
+	flux module load ${TEST_WATCHER} &&
```

Comment:
shouldn't there be a `check_kvs_watchers` call after loading the module, to reduce race possibilities?

Reply:
Oh I thought I had that covered because the RPC was sent before the reactor was entered, but you're right of course!  We don't know if the request has made it to the KVS by the time we check.